src/runtime/polyfillsModules.js

```diff
@@ -0,0 +1,27 @@
+// Copyright 2015 Traceur Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/** @fileoverview import all polyfills, eg for Traceur self-build. */
+
+import './polyfills/Map.js';
```

Comment:
Yes, it sucks. But the right solution is to remove the Makefile. The list is used to track dependencies for the build, otherwise we have no way to know that we should rebuild if the individual polyfill files change.